Releasing.md

```diff
@@ -0,0 +1,152 @@
+## Generating the licenses for the NPM dependencies
+
+Before a release can be performed the npm license need to be updated with any new dependencies. This file is found under `artemis-console-distribution/src/main/resources/licenses/NPMLicenses.txt`.
+
+This is done using the NPM [yarn-plugin-licenses](https://github.com/mhassan1/yarn-plugin-licenses) plugin, you will need to install this if you havent already.
```

Comment:
The .yarnrc file has an entry for this, so is this true at this point?

Reply:
removed